CommonControls/PackFileBrowser/ContextMenuHandler.cs

```diff
@@ -127,11 +134,44 @@ void OnAddFilesCommand()
                 foreach (var file in files)
                 {
                     var fileName = Path.GetFileName(file);
+
                     var packFile = new PackFile(fileName, new MemorySource(File.ReadAllBytes(file)));
+
                     _packFileService.AddFileToPack(_selectedNode.FileOwner, parentPath, packFile);
                 }
             }
         }
+        void OnImport3DModelCommand()
+        {
+            var parentPath = _selectedNode.GetFullPath(); // get pack path, at mouse pointer        
```

Comment:
All this logic from the "var dialog = new CommonOpenFileDialog();" line should go in the new ImportService

Reply:
this is not done